core/src/main/java/com/es/core/model/phone/JdbcPhoneDao.java

```diff
@@ -13,15 +13,33 @@
     @Resource
     private JdbcTemplate jdbcTemplate;
 
+    public void setJdbcTemplate(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
     public Optional<Phone> get(final Long key) {
-        throw new UnsupportedOperationException("TODO");
+        return Optional.ofNullable(jdbcTemplate.queryForObject("select * from phones where id=?",
```

Comment:
You forgot to include color, also change all other requests. You will need to include some logic to map results from ResultSet, if it will be some custom row mapper, please than use it as spring bean